test/lit/wat-kitchen-sink.wast

```diff
@@ -466,15 +476,9 @@
  )
 
  ;; CHECK:      (func $add-twice-unreachable (type $ret2) (result i32 i32)
- ;; CHECK-NEXT:  (tuple.make
- ;; CHECK-NEXT:   (i32.add
- ;; CHECK-NEXT:    (unreachable)
- ;; CHECK-NEXT:    (i32.const 2)
- ;; CHECK-NEXT:   )
- ;; CHECK-NEXT:   (i32.add
- ;; CHECK-NEXT:    (i32.const 3)
- ;; CHECK-NEXT:    (i32.const 4)
- ;; CHECK-NEXT:   )
+ ;; CHECK-NEXT:  (i32.add
+ ;; CHECK-NEXT:   (unreachable)
+ ;; CHECK-NEXT:   (i32.const 2)
```

Comment:
Looks correct, but is this still testing all we wanted it to test earlier?

Reply:
Yep, these tests are only trying to check that the parse is correct in the sense that it has the same semantics as the input.